route/route.go

```diff
@@ -73,9 +73,7 @@ func (r *Route) addTarget(service string, targetURL *url.URL, fixedWeight float6
 		t.TLSSkipVerify = r.Opts["tlsskipverify"] == "true"
 		// If the 'usehost' key exists in the map, we set
```

Comment:
Can you drop the comment, too please? Don't want to be overly pick but it really is redundant.